.github/workflows/android-publish.yml

```diff
@@ -0,0 +1,158 @@
+      MIMOTO_HOST:
+        description: "Mimoto backend service URL"
+        required: true
+        default: "https://api.sandbox.mosip.net"
+        type: string
+      ESIGNET_HOST:
+        description: "Esignet backend service URL"
+        required: true
+        default: "https://api.sandbox.mosip.net"
+        type: string
+      APPLICATION_THEME:
+        description: "Application Theme"
+        required: true
+        default: "orange"
+        type: choice
+        options:
+          - orange
+          - purple
+      BUILD_DESCRIPTION:
+        description: "What to test"
+        required: true
+        default: "QA-Triple environment"
+        type: string
+      ALLOW_ENV_EDIT:
+        description: "Edit ENV"
+        required: true
+        default: "true"
+        type: choice
+        options:
+          - false
+          - true
+      BUILD_NAME:
+        description: "Build App For"
+        required: true
+        default: "Sprint-x/Collab/release-x.x.x"
+        type: string
+    secrets:
+      ANDROID_KEYSTORE_FILE:
+        required: true
+      RELEASE_KEYSTORE_PASSWORD:
+        required: true
+      INJI_ANDROID_PLAY_STORE_CONFIG_JSON:
+        required: true
+      ENV_FILE:
+        required: true
+      SLACK_URL:
+        required: true
+
+jobs:
+  deploy-android:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3.1.0
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ inputs.NODE_VERSION }}
+
+      - name: Check for git tag
+        if: ${{ inputs.BETA_RELEASE == 'Yes' }}
+        run: |
+          cd android/scripts
+          ./git-tag.sh
+
+      - name: Install npm dependencies
+        run: npm ci
+
+      - name: Create .env.local file
+        run: |
+          echo "${{ secrets.ENV_FILE }}" > .env.local > android/local.properties
+          echo "BRANCH_NAME=$(echo ${{ github.ref }} | sed -e 's,.*/\(.*\),\1,')" >> $GITHUB_ENV
+
+      - name: Generate keystore
+        run: |
+          echo "$ANDROID_KEYSTORE_FILE" > release.keystore.b64
+          base64 -d -i release.keystore.b64 > android/app/release.keystore
+        env:
+          ANDROID_KEYSTORE_FILE: ${{ secrets.ANDROID_KEYSTORE_FILE }}
+
+      - name: Create Build
+        run: |
+          cd android/scripts
+          ./verify-build.sh
+        env:
+          MIMOTO_HOST: ${{ inputs.MIMOTO_HOST }}
+          ESIGNET_HOST: ${{ inputs.ESIGNET_HOST }}
+          APPLICATION_THEME: ${{ inputs.APPLICATION_THEME }}
+          RELEASE_KEYSTORE_ALIAS: ${{ inputs.RELEASE_KEYSTORE_ALIAS }}
+          RELEASE_KEYSTORE_PASSWORD: "${{secrets.RELEASE_KEYSTORE_PASSWORD}}"
+          CREDENTIAL_REGISTRY_EDIT: ${{ inputs.ALLOW_ENV_EDIT }}
+
+      - name: Upload Artifact to Actions
+        uses: actions/upload-artifact@v3.1.1
+        with:
+          name: ${{ inputs.BUILD_NAME }}
+          path: android/app/build/outputs/apk/mosip/release/Inji_universal.apk
+          retention-days: 10
```

Comment:
This is required for only android-build.yml file not for android-publish

Reply:
We are using this build file only to verify whether the app is successfully built. We are uploading the apk as part of publish workflow, where we need apk for the uploaded bundle in play console.